src/MDMPerforming.h

```diff
@@ -114,3 +114,31 @@ NS_SWIFT_NAME(ComposablePerforming)
     NS_SWIFT_NAME(set(transactionEmitter:));
 
 @end
+
+/** Specifics for a named plan performer to allow named plans to be added and removed. */
+NS_SWIFT_NAME(NamedPlanPerforming)
+@protocol MDMNamedPlanPerforming <MDMPerforming>
+
+@optional
```

Comment:
In the `Performer Specification` section at https://material-motion.gitbooks.io/material-motion-starmap/content/specifications/runtime/named-plans.html, the spec mentions the following for add/remove operations:
```
If one method is implemented, the other must be implemented as well
```
I'm not sure there's a viable way to enforce this type of rule with protocols in Objective-C, so I made the two methods optional. Perhaps they should both be required?